qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -85,7 +85,9 @@ def run(self, qobj: Qobj, job_name: Optional[str] = None) -> IBMQJob:
         kwargs = {}
         if isinstance(self._api, BaseClient):
             # Default to using object storage and websockets for new API.
-            kwargs = {'use_object_storage': True,
+
+            use_object_storage = self._configuration['allow_object_storage']
```

Comment:
Note: `allow_object_storage` may not be the actual attribute name. It is simply a placeholder as of right now. It will be changed once the attribute is added to the backend configurations.

Reply:
Seems it is the final name!